src/cmd/root.go

```diff
@@ -24,7 +45,13 @@ func Execute() {
 func init() {
 	rootCmd.AddCommand(checkCmd)
 	rootCmd.AddCommand(checkFileCmd)
+	rootCmd.AddCommand(newRulesetCommand)
+	rootCmd.AddCommand(generateRuleCommand)
+	rootCmd.AddCommand(generateShortCommand)
 
 	checkFileCmd.Flags().Bool("gui-child-mode", false, "Tells app that it is executed by gui")
 	checkCmd.Flags().Bool("gui-child-mode", false, "Tells app that it is executed by gui")
+	newRulesetCommand.Flags().Bool("gui-child-mode", false, "Tells app that it is executed by gui")
+	generateRuleCommand.Flags().Bool("gui-child-mode", false, "Tells app that it is executed by gui")
+	generateShortCommand.Flags().Bool("gui-child-mode", false, "Tells app that it is executed by gui")
```

Comment:
I don't think it is needed for your command, because I created this flag only in order to redirect all api.info.* to dbus